### PR TITLE
feat(ASB-0017): implement automatic assets creation

### PR DESCRIPTION
**Pull request by zdevaty**
Except doorlocks, for now.

### CODE REVIEW

File: conf/conf.go [hunk 1]
```diff
@@ -97,6 +97,15 @@ func GetAssetId(ctx context.Context, config apiserver.Configuration, projId stri
 	return common.Ptr(int32(dbAssets[0].ID)), nil
 }
 
+func InsertAsset(ctx context.Context, config apiserver.Configuration, projId string, SerialNumber string, assetId int32) error {
+	var dbAsset dbkentix.Sensor
```
Comment (andreas-goebel) [inline conf/conf.go]: Maybe InsertSensor is the better function name. InsertAsset implies to save an asset in Eliona.

Comment (andreas-goebel) [reply conf/conf.go]: Then dbAsset should be dbSensor

File: eliona/assets.go [hunk 2]
```diff
@@ -85,3 +85,61 @@ func name(specification kentix.DeviceInfo) string {
...
+	name := doorlockName(spec)
+	description := doorlockDescription(spec)
+
+	newId, err := asset.UpsertAsset(api.Asset{
+		ProjectId:               projectId,
+		GlobalAssetIdentifier:   spec.Serial,
+		Name:                    *api.NewNullableString(common.Ptr(name)),
+		AssetType:               kentix.DoorlockAssetType,
+		Description:             *api.NewNullableString(common.Ptr(description)),
+		ParentLocationalAssetId: *api.NewNullableInt32(parentAssetId),
```
Comment (andreas-goebel) [inline eliona/assets.go]: There are 2 types of relationship: Local and functional. I think, in this case the functional relationship is the better one -> parentFunctionalAssetId

Some other words to illustrate the difference: **local** = geographic positions e.g. a door  and a room. You can not change the place of the door in relation to the room. **functional** = logical dependency e.g. door and keycard to open it. The keycard's position is not fix, but it belongs to the door.

For Kentix: the doorlock should have the AccessManager as functional parant. In addition the local parent can be a Room asset.

**Review by andreas-goebel — CHANGES_REQUESTED** [review]

File: eliona/assets.go (hunk 2 above)
Comment (zdevaty) [reply eliona/assets.go]: Aha, thanks for the explanation.

**Review by andreas-goebel — APPROVED** [review]

**Pull request merged**